src/test/java/com/mongodb/jdbc/MongoSQLResultSetMetaDataTest.java

```diff
@@ -20,7 +20,7 @@
 
     static {
         try {
-            resultSetMetaData = new MongoSQLResultSetMetaData(generateMongoJsonSchema());
+            resultSetMetaData = new MongoSQLResultSetMetaData(generateMongoJsonSchema(), true);
```

Comment:
How did you determine `true` vs `false` here? Should we test both via unit testing?

Reply:
Maybe I just misunderstood what this is for. I checked the name and the functions and thought it was for validation resultset metadata for database metadata. I'll double-check on Monday.

Reply:
You are right, both classes are related to validating resultset from "queries", not from database metdata result sets. Used "true" for both.